frontend/src/__tests__/integration/components/pipelines/PipelineSelector.stories.tsx

```diff
@@ -24,6 +25,7 @@ export const Default: StoryObj = {
     // load page and wait until settled
     const canvas = within(canvasElement);
     await canvas.findByText('Select a pipeline version', undefined, { timeout: 5000 });
+    await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
Actually, I can remove this delay, it won't break the test lol. It will only cause a small layout issue in the storybook but it doesn't matter. I will remove it.